### PR TITLE
Support system property configuration

### LINKED ISSUE #106: Accept systems properties as well as environment variables
Currently ErisCasper accepts configuration via environment variables (e.g., `EC_TOKEN=my.bot.token`).

Java projects often respect system properties as well (e.g., `-Dec.token=my.bot.token`)

We should accept either. If both are given, prioritize the environment variable setting.

### PR DESCRIPTION
**Pull request by ianagbip1oti**
Fixes #106

### CODE REVIEW

File: src/main/java/com/github/princesslana/eriscasper/ErisCasper.java [hunk 1]
```diff
@@ -57,7 +68,10 @@ public void run(Bot bot) {
   }
 
   public static ErisCasper create() {
-    return create(System.getenv("EC_TOKEN"));
+    return create(
+        CONFIG
+            .get("ec.token")
```
Comment (CoreyShupe) [inline src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: How does this work? Should we throw in a default configuration file for example. I think this is a bit ambiguous as to what to do. Maybe throw a config file into the resources and if not built create a default one for them. Something needs to be done so before installation there's a way to figure out how to set it up, rather that be in docs or some other example.

**Review by CoreyShupe — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/princesslana/eriscasper/ErisCasper.java (hunk 1 above)
Comment (ianagbip1oti) [reply src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: This doesn't read from files yet, it just adds system properties as per #106. So you can configure the token by setting the `EC_TOKEN` environment variable, or the `ec.token` system property.

We do need a getting started guide, but imo that's outside the scope of this issue.

Comment (CoreyShupe) [reply src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: Ah okay, so this is just the beginning to implementing a configuration file, or rather the implementation to provide support for it.

Comment (ianagbip1oti) [reply src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: It's the beginning of enabling configuration to happen in ways other than just environment variables, yes.

Comment (CoreyShupe) [reply src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: Then this is probably good for now, it definitely needs work in documentation and examples for installation. I'll put that in the issue list.

Comment (ianagbip1oti) [reply src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: #108 exists for documentation

Comment (CoreyShupe) [reply src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: I don't see configuration specification. Should we add that as a checklist item.

Comment (ianagbip1oti) [reply src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: Or leave a comment

Comment (CoreyShupe) [reply src/main/java/com/github/princesslana/eriscasper/ErisCasper.java:73]: Alright, I'll leave a comment there and accept this PR through.

**Review by CoreyShupe — APPROVED** [review]

**Pull request merged**